keps/sig-network/2004-topology-aware-subsetting/kep.yaml

```diff
@@ -26,7 +26,7 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.21"
+# latest-milestone: "v1.21"
```

Comment:
@ckyiricky  @robscott  FYI, PRR approval will need to be part of the PR that sets this back to 1.21.